bentoml/lightgbm.py

```diff
@@ -230,7 +243,9 @@ def load_runner(
 
     Args:
         tag (`str`):
-            Model tag to retrieve model from modelstore
+            Model tag to retrieve model from modelstore.
+        predict_fn_name (`str`, default to `"predict"`):
```

Comment:
Lets change `predict_fn_name` to `infer_api_callback`, and the doc would be
```
infer_api_callback (`str`, `optional`, default to `predict`):
	Inference API callback from given model. If not specified, BentoML will use default `predict`. Users can also choose to use `predict_proba` for supported model
```

Reply:
https://github.com/bentoml/BentoML/tree/bentoml-1.0/docs We are trying to enforce our docs format to follow like so :)

Reply:
It is ok to keep it like this, you just have to add the tests for this, I will merge it and then fix it  for you if you want to.

Reply:
don't worry! i'll add the test cases asap~